tools/test-js-with-node

```diff
@@ -298,6 +298,7 @@ EXEMPT_FILES = make_set(
         "web/src/zform.js",
         "web/src/zulip.js",
         "web/src/zulip_test.ts",
+        "web/tests/lib/example_user.js",
```

Comment:
Currently the `make_cross_realm_bot` function lacks coverage, since the example tests did not have a use-case for `cross_realm_bot`s.